realsense2_camera_msgs/package.xml

```diff
@@ -16,10 +16,12 @@
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
   <build_depend>builtin_interfaces</build_depend>
   <build_depend>std_msgs</build_depend>
+  <build_depend>sensor_msgs</build_depend>
```

Comment:
But rgbd is a custom message, right?
I think the camera depend on camera_msgs
and it may have this depends?

Reply:
this is the camera_msgs package.xml, so it should be here.
there was another duplication inside "realsense2_camera" package.xml that I removed

Reply:
Got it.